source/_docs/guides/circleci.md

```diff
@@ -0,0 +1,235 @@
+
+    ```nohighlight
+    git remote add origin git@github.com:YOUR-ORG/YOUR-PROJECT
+    git push -u origin master
+    ```
+
+3. Once the push is complete on your local, reload the browser to see your files in the GitHub repository:
+
+    ![Initial commit to new repo](/source/docs/assets/images/integrations/first_commit.png)
+
+4. Create a new [GitHub access token](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/). Keep the token handy so we can use it on CircleCI later. Be sure to give it the "repo" or "private repo" scope to match the privacy settings for the GitHub repository.
+
+## Run the Project on Pantheon
+Now we will spin up a Drupal 8 site, then overwrite the default installation with code from our GitHub repository.
+
+1. From your Pantheon User or Organization Dashboard, [create a new Drupal 8 site](/docs/create-sites) and click **Visit your Pantheon Dashboard**.
+
+2. Click **Visit Development Site** and complete the Drupal 8 installation process. Keep the Site Name, Admin email and password handy so we can use it on CircleCI later.
+
+3. Commit your `settings.php` changes to the Dev environment from the Site Dashboard, then change the connection mode to **Git**.
+
+4. Under Connection Info, copy the **SSH clone URL**:
+
+    ![SSH Clone URL from dashboard](/source/docs/assets/images/integrations/ssh_url.png)
+
+5. Paste the URL into a local text editor, and copy the Git repository URL; it's the middle part which begins `ssh://` and ends with `.git`:
+
+    ![SSH Clone URL Detail](/source/docs/assets/images/integrations/ssh_url_det.png)
+
+6. With that URL, create and run the following Git command in the root directory of your local project:
+
+    ```bash
+    git remote add pantheon ssh://codeserver...drush.in:2222/~/repository.git
+    ```
+
+    This will create a second [remote](https://git-scm.com/docs/git-remote), called `pantheon`.
+
+7. Push your code to the `pantheon` remote:
+
+    ```bash
+    git push --force pantheon master
+    ```
+
+    In the terminal, you should see Pantheon applying the pantheon.yml file configuration:
+
+    ``` bash
+    remote: PANTHEON NOTICE:
+    remote:
+    remote: Changes to `pantheon.yml` detected.
+    remote:
+    remote: Successfully applied `pantheon.yml` to the 'dev' environment.
+    ```
+
+    When this is complete, the site should have a single commit, and be in sync with the GitHub repo.
+
+    ![Updated Site Dashboard](/source/docs/assets/images/integrations/updated.png)
+
+## Create Machine User and Token on Pantheon
+Next, we will create a new Pantheon machine user to associate with our CircleCI implementation.
+
+1. Using a unique email address, create a new Pantheon user from the [Pantheon home page](https://pantheon.io/) and log in. For example, this site uses `docs@pantheon.io` for our machine user.  
+
+2. [Create an SSH key](/docs/ssh-keys) for this user. Be careful not to overwrite any existing SSH keys.
```

Comment:
Can we add context to the warning here? Maybe add a warning div element and give instructions on how to generate a key without overwriting the current/default/active key in the ssh agent.